templates/zerver/integrations/google-calendar.md

```diff
@@ -1,72 +1,64 @@
-Get Google Calendar reminders in Zulip! This is a great way to see
+# Zulip Google Calendar integration
+
+Get Google Calendar reminders in your Zulip DMs! This is a great way to see
```

Comment:
"Get Google Calendar reminders in Zulip" -> "Get Google Calendar reminders in your Zulip DMs"